data_model/derive/src/lib.rs

```diff
@@ -23,55 +23,74 @@ mod id;
 /// As a rule of thumb, this derive should never be used on any structs that can't be uniquely identifiable,
 /// as all the derived traits here rely on the fact of that uniqueness.
 ///
-/// Example:
-/// ```rust
+/// # Examples
 ///
-/// // For a struct decorated like this
-/// #[derive(IdOrdEqHash)]
+/// ```
```

Comment:
```suggestion
/// ```rust
```

Reply:
Implied. And I don't see notations like `/// ```rust,ignore` in the rustdoc book

Reply:
Ok, `/// ```rust` may be useful for grep